HiP-DataStore.Model/Rest/TagUpdateArgs.cs

```diff
@@ -0,0 +1,22 @@
+﻿using MongoDB.Bson.Serialization.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PaderbornUniversity.SILab.Hip.DataStore.Model.Rest
+{
+    public class TagUpdateArgs
```

Comment:
I discussed updates and PUT requests with Patrick (see latest comments on page "REST API for mobile content" on Confluence). We came to the conclusion that PUT requests shouldn't be used to update individual fields of an entity, but rather to replace an entity completely. Thus, PUT requests should not only specify values for fields that should be changed, but also for fields that do not change. In other words, POST and PUT always have exactly the same kinds of parameters, so TagUpdateArgs is redundant and we can just use TagArgs instead.